layers/meta-balena-raspberrypi/recipes-bsp/bootfiles/rpi-config_git.bbappend

```diff
@@ -28,6 +28,12 @@ do_deploy:append:revpi-connect() {
 	echo "dtoverlay=revpi-connect" >> ${DEPLOYDIR}/bootfiles/config.txt
 }
 
+do_deploy:append:revpi-connect-s() {
+	# Use the RevPi Connect device tree overlay
+	echo "dtoverlay=revpi-connect" >> ${DEPLOYDIR}/bootfiles/config.txt
```

Comment:
Does the "revpi connect s" use the exact same overlay as the older "revpi connect"?

Reply:
Yes it is using the same overlay as the REVPI Connect. should I change this?

Reply:
If it is using the exact same overlay then it's fine. I thought there may be a specific overlay for the S version.